website/docs/src/content/component-reference/storage-components/blob-reference.mdx

```diff
@@ -0,0 +1,383 @@
+---
+title: Blob storage reference
+description: Blob storage components for stateful workflows and agents.
+---
+
+# Blob storage reference
+
+API reference for GenSX Cloud blob storage components.
+
+## Installation
+
+```bash
+npm install @gensx/storage
+```
+
+## BlobProvider
+
+Provides blob storage capabilities to its child components.
+
+### Import
+
+```tsx
+import { BlobProvider } from "@gensx/storage";
+```
+
+### Props
+
+| Prop            | Type                      | Default        | Description                           |
+| --------------- | ------------------------- | -------------- | ------------------------------------- |
+| `kind`          | `"filesystem" \| "cloud"` | Auto-detected  | Storage backend to use                |
```

Comment:
we should add a sentence somewhere (maybe line 18?) saying how it's autodetected